orc8r/gateway/python/magma/ctraced/trace_manager.py

```diff
@@ -152,22 +237,18 @@ def _stop_trace(self) -> bool:
             logging.info("TraceManager: Tracing process return code: %s",
                          return_code)
 
-        logging.debug("TraceManager: Trace logs:")
-        logging.debug("<" * 25)
-        while True:
-            line = self._proc.stdout.readline()
-            if not line:
-                break
-            logging.debug("| %s", str(line.rstrip()))
-        logging.debug(">" * 25)
+        self._dump_trace_logs()
 
         if return_code is not None:
             self._is_active = False
             return False
 
         return True
 
-    def _postprocess_trace(self) -> bool:
+    def _conditionally_postprocess_trace(self) -> bool:
+        if len(self._display_filters) == 0:
```

Comment:
len(self._display_filters) -> self._display_filters is None

Reply:
It's always a string

Reply:
It's always a string